yt_dlp/extractor/bitchute.py

```diff
@@ -42,12 +47,7 @@ def _real_extract(self, url):
                 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.57 Safari/537.36',
             })
 
-        title = self._html_search_regex(
-            (r'<[^>]+\bid=["\']video-title[^>]+>([^<]+)', r'<title>([^<]+)'),
-            webpage, 'title', default=None) or self._html_search_meta(
-            'description', webpage, 'title',
-            default=None) or self._og_search_description(webpage)
-
+        title = self._html_extract_title(webpage) or self._og_search_title(webpage)
```

Comment:
This is not just a refactor, but change where title is extracted from. Is that intentional?

Reply:
Yes. The old implementation is 4 years old. Bitchute's HTML might change over time, but is equal for all videos.
And if there is a corner case one should provide a test for it.